frontend/app/src/composables/balances/breakdown.ts

```diff
@@ -24,10 +24,10 @@ export function useBalancesBreakdown(): UseBalancesBreakdownReturn {
   const { isAssetIgnored } = useIgnoredAssetsStore();
   const { toSortedAssetBalanceWithPrice } = useBalanceSorting();
 
-  const assetBreakdown = (asset: string): ComputedRef<AssetBreakdown[]> => computed<AssetBreakdown[]>(() =>
+  const assetBreakdown = (asset: string, isLiability = false): ComputedRef<AssetBreakdown[]> => computed<AssetBreakdown[]>(() =>
     groupAssetBreakdown(
-      get(getBreakdown(asset))
-        .concat(get(getManualBreakdown(asset)))
+      get(getBreakdown(asset, isLiability))
+        .concat(get(getManualBreakdown(asset, isLiability)))
         .concat(get(getExchangeBreakdown(asset)))
```

Comment:
wouldn't this change mix exchange assets and manual/blockchain liabilities?

I think it would be better to use separate methods instead of flags to track liabilities instead.